app-config.yaml

```diff
@@ -352,8 +352,6 @@ auth:
         metadataUrl: ${AUTH_OIDC_METADATA_URL}
         clientId: ${AUTH_OIDC_CLIENT_ID}
         clientSecret: ${AUTH_OIDC_CLIENT_SECRET}
-        authorizationUrl: ${AUTH_OIDC_AUTH_URL}
-        tokenUrl: ${AUTH_OIDC_TOKEN_URL}
```

Comment:
I could not find these any where, if they were used I belive they would have been mentioned here: https://github.com/backstage/backstage/blob/1a14adeb574f7846b6d5f5ef555a57dea403e03a/plugins/auth-backend/src/providers/oidc/provider.ts#L231